dashboards-observability/public/components/visualizations/charts/bar/bar.tsx

```diff
@@ -48,7 +54,7 @@ export const Bar = ({ visualizations, layout, config }: any) => {
    * determine stylings
    */
   const barOrientation = dataConfig.chartStyles?.orientation || visMetaData.orientation;
-  const isVertical = barOrientation === visMetaData.orientation;
+  const isVertical = barOrientation === 'v';
```

Comment:
Please use a constant variable for this, never use a direct string value.

Reply:
done